README.md

```diff
@@ -44,8 +44,8 @@ Make sure your web.xml is [configured to expose your endpoints](https://cloud.go
 
 No configuration parameters are required to run with default values
 ```
```

Comment:
nit: I think 'shell' might be a good language directive here for possible syntax highlighting and $ can be removed.